firmware/console/binary/tunerstudio_io.cpp

```diff
@@ -245,20 +242,34 @@ void sr5WriteCrcPacket(ts_channel_s *tsChannel, const uint8_t responseCode, cons
 	}
 #endif /* TS_CAN_DEVICE */
 
-	*(uint16_t *) writeBuffer = SWAP_UINT16(size + 1);   // packet size including command
-	*(uint8_t *) (writeBuffer + 2) = responseCode;
+	auto scratchBuffer = tsChannel->crcReadBuffer;
```

Comment:
shall we rename 'crcReadBuffer' since at this point that's not really a 'read' buffer?

Reply:
good idea!